### PR TITLE
[Docker] Commit changes on docker

### PR DESCRIPTION
**Pull request by barry-jin**
## Description ##
Commit changes after updating the docker by setting MXNet to latest version in AWS Batch CI. 

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

cc @dmlc/gluon-nlp-team

### CODE REVIEW

File: tools/docker/gluon_nlp_job.sh [hunk 1]
```diff
@@ -23,14 +23,14 @@ git fetch origin $SOURCE_REF:working
 git checkout working
 
 if [ $DEVICE == "cpu" ]; then
-  python3 -m pip install -U --quiet --pre "mxnet==2.0.0b20201016" -f https://dist.mxnet.io/python --user
+  python3 -m pip install -U --quiet --pre "mxnet>=2.0.0b20200802" -f https://dist.mxnet.io/python --user
```
Comment (sxjscience) [inline tools/docker/gluon_nlp_job.sh:26]: Why 0802?

**Comment by sxjscience** [comment]
How about to merge this PR with the doc improvement PR?

**Comment by barry-jin** [comment]
I have just merged this branch to doc improvement branch.

**Comment by sxjscience** [comment]
@barry-jin So let me close this PR.

**Pull request closed without merging**